### PR TITLE
shows the error handling tests completed as well as a search bug fix

### PR DESCRIPTION
**Pull request by JacobMacFarlane**
____ Wrote Tests ____ Implemented ____ Reviewed

## Neccesary checkmarks
- [ X] All Tests are Passing in all environments
- [ X] The code will run locally
## Type of change
- [ X] New feature
- [X ] Bug Fix
- [ ] Refactor
## Description of Change: Added testing for the following
Bad fetch call on Home page
Bad Fetch call on vendor view
Bad Search results which actually led to a bug fix within the reset results function for the search
Bad URL route
these are all the spots where i could for see error handling needed

    
## Requested feedback:  
    
  I'd like feedback on if i missed any obvious error handling or bad routes please let me know

## Check the correct boxes

- [ X] This broke nothing
- [ ] This broke some stuff
- [ ] This broke everything

## Testing Changes

- [ ] No Tests have been changed
- [X ] Some Tests have been changed
- [ ] All of the Tests have been changed(Please describe what in the world happened)

## Checklist:

- [ ] My code has no unused/commented out code
- [ ] My code has no binding.pry's
- [ X] I have reviewed my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [X ] I have fully tested my code

(For Fun!)Please Include a link to a gif of your feelings about this branch

Link:

### CODE REVIEW

File: src/Components/App/App.js [hunk 1]
```diff
@@ -59,7 +59,7 @@ const App = () => {
     let lowerSearchValue = searchValue.toLowerCase()
     let nameSearchResults = vendors.filter((v) => v.name.toLowerCase().includes(lowerSearchValue) || v.tags.toLowerCase().includes(lowerSearchValue) || v.description.toLowerCase().includes(lowerSearchValue))
     if (!nameSearchResults.length) {
-      setVendors(storedVendors)
+      // setVendors(storedVendors)
```
Comment (Devynnem) [inline src/Components/App/App.js:62]: Good bug check!

**Review by Devynnem — APPROVED** [review]

File: src/Components/App/App.js [hunk 2]
```diff
@@ -90,13 +90,14 @@ const App = () => {
   const resestResults = (event) => {
     setError('')
     event.preventDefault()
-    setVendors(storedVendors)
+    console.log(vendors, 'vendors after rest')
+    console.log(storedVendors, 'vendors after rest')
```
Comment (Pma913) [inline src/Components/App/App.js:94]: Did you mean to leave these console logs in?

Comment (JacobMacFarlane) [reply src/Components/App/App.js:94]: No i did not

Comment (Pma913) [reply src/Components/App/App.js:94]: can you clear them and push again?

Comment (JacobMacFarlane) [reply src/Components/App/App.js]: Done

**Review by Pma913 — APPROVED** [review]
thanks for removing those logs!

**Pull request merged**